cabot/metricsapp/tests/test_elasticsearch.py

```diff
@@ -6,7 +6,8 @@ class TestElasticsearchSource(TestCase):
     def setUp(self):
         self.es_source = ElasticsearchSource.objects.create(
             name='elastigirl',
```

Comment:
it's from the incredibles http://disney.wikia.com/wiki/Helen_Parr
sorta close to elasticsearch.....